docs/semgrep-code/getting-started.md

```diff
@@ -29,333 +28,146 @@ import PlatformSigninGithub from "/src/components/procedure/_platform-signin-git
-:::
+## Scan a repository on your local machine
 
-#### Option A: Scanning a local repository through Semgrep CLI
+You can perform a one-time scan of your repository locally using the Semgrep CI. To
+do so:
 
-Scanning a project from the CLI is a standalone action. This means that you manually triggered the scan. Scans from CLI are not continuous nor scheduled, unlike scans run in a CI job.
+1. Log in to Semgrep Cloud Platform.
 
-:::info Prerequisites
-- Semgrep CLI must be installed. See [Getting started with Semgrep OSS Engine](/getting-started/quickstart/).
-- Ensure that you are running a recent version of Semgrep CLI. Semgrep Cloud Platform supports the 10 most recent minor versions of Semgrep CLI.
-:::
+2. Navigate to **Projects** > **Scan new project**.
 
-To scan a local repository through Semgrep CLI, follow these steps:
+3. Select **Locally**, and follow the instructions displayed on the **Run a scan
+   locally** screen. For your convenience, we've reproduced that information
+   here:
 
-1. Ensure you are signed in to Semgrep Cloud Platform.
-2. Click **[Projects](https://semgrep.dev/orgs/-/projects)** on the left sidebar.
-3. Click **Scan new project** > **Run a scan locally**.
-4. Log in to Semgrep from the CLI:
-    ``` 
+    ```console
+    # log in to Semgrep
     semgrep login
-    ```
-5. Follow the instructions in the CLI.
-6. After logging in, run a scan by entering the following command. This command sends the findings to Semgrep Cloud Platform.
-    ```
+
+    # run the scan
     semgrep ci
     ```
-7. View your project's [findings](https://semgrep.dev/orgs/-/findings).
 
-:::note
-Scans from local repositories do not access their corresponding remote repositories. For this reason, links to specific lines of code in the Findings page are not created. See [Linking local scans to their remote repositories](#linking-local-scans-to-their-remote-repositories) for a workaround.
-:::
-
-##### Linking local scans to their remote repositories 
-
-![Screenshot of findings page snippet with no hyperlinks](/img/findings-no-hyperlinks.png "Screenshot of findings page snippet with no hyperlinks")
-*Figure 1.* Partial screenshot of findings page with no hyperlinks.
+4. Once the scan completes, return to Semgrep Cloud Platform and click **View
+   findings** to see your results. Alternatively, you can go directly to your
+   [findings](https://semgrep.dev/orgs/-/findings) page.
 
-Local repository scans require additional configuration to **create hyperlinks to their corresponding remote repositories**. Set up environment variables within your command line to configure cross-linking between local and remote repositories within Semgrep Cloud Platform.
+### Link local scans to their remote repositories 
```

Comment:
This section is pretty useful for certain use cases, especially as a fallback for any misconfigurations in `semgrep ci`, but I think we'll have to move it out of this doc - it's not a golden path we want to user to have to wade through. We probably need to add to this PR a dedicated guide to running `semgrep ci` in a local CLI, not a remote environment. I realize it's a very big PR at this point, that's why we're refactoring it.

I can commit directly to this PR to address this, if you're ok with it, without touching any of the suggestions I'm making. LMK if that works for you.

Reply:
> I can commit directly to this PR to address this, if you're ok with it, without touching any of the suggestions I'm making. LMK if that works for you.

Yes, that's perfectly fine with me!

Reply:
Update: Sara to make follow up PR instead so we don't add too much to this one and just approve it.